src/views/Article.vue

```diff
@@ -78,7 +78,20 @@ export default {
       return false;
     },
     updateTitle() {
-
+    },
+    compiledMarkdown() {
+      return markdownIt.render(this.post.content);
+    },
+    getClipboard() {
+      const {currentUsername, scatterAccount} = this;
+      // todo(minakokojima): figure out what is the different between following variables.
```

Comment:
no difference here
`currentUsername` is the projection of `scatterAccount.name` which you can find out [here](https://github.com/smart-signature/smart-signature-future/blob/4b618f04928eee504e26614d9c4a8135ea5e8afc/src/store.js#L21)
```js
currentUsername: ({ scatterAccount }) => (scatterAccount ? scatterAccount.name : null),
```